src/stream_wrap.cc

```diff
@@ -239,8 +239,7 @@ static MaybeLocal<Object> AcceptHandle(Environment* env,
   return scope.Escape(wrap_obj);
 }
 
-
-void LibuvStreamWrap::OnUvRead(ssize_t nread, const uv_buf_t* buf) {
+v8::Maybe<void> LibuvStreamWrap::OnUvRead(ssize_t nread, const uv_buf_t* buf) {
```

Comment:
```suggestion
Maybe<void> LibuvStreamWrap::OnUvRead(ssize_t nread, const uv_buf_t* buf) {
```
nit: could we create using declarations for the newly used V8 symbols and use them without the `v8::` prefix just for this file?

Reply:
Done!